build/Helix/CopyVisualTreeVerificationFiles.ps1

```diff
@@ -0,0 +1,2 @@
+$picturesFolder = [Environment]::GetFolderPath('MyPictures')
```

Comment:
visual tree verification necessary, vestigial, or "keep because it helps us figure out diffs from upstream helix"?